holidays/locale/en_US/LC_MESSAGES/TC.po

```diff
@@ -0,0 +1,76 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# Turks and Caicos Islands holidays en_US localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.22\n"
+"POT-Creation-Date: 2023-03-23 14:44+0200\n"
+"PO-Revision-Date: 2023-03-23 14:47+0200\n"
+"Last-Translator: Your Name <your.email@example.com>\n"
```

Comment:
Here should be version 0.71 (currently), the latest dates, and your name and email.